.eleventy.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function (eleventyConfig) {
+  eleventyConfig.addPassthroughCopy("assets");
```

Comment:
`addPassthroughCopy` allows us to move over any assets or directories we need to the final site build.